ci/generate_exclude_filter.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+
+# NOTE: This script requires Python >= 3.7 and CMake >= 3.15
```

Comment:
Should we bump CMake requirements?
https://github.com/KhronosGroup/SYCL-CTS#compilation-and-execution-prerequisites requires CMake 3.13.

Reply:
This script also requires ninja

Reply:
I'm okay to update prerequisites listed in README file with follow-up PR(s).